File: src/main/java/ru/vaschenko/ParkPoint/dto/ParkingZoneDto.java
package ru.vaschenko.ParkPoint.dto;

import java.util.List;

import ru.vaschenko.ParkPoint.enams.StateParkingZone;
import ru.vaschenko.ParkPoint.models.Photo;

public record ParkingZoneDto(
    Long id,
    Long zoneManager,
    String title,
    String address,
    String description,
    String map,
    StateParkingZone state,
    int parkingSpacesCount,
    double latitude,
    double longitude,
    List<Photo> photos) {
}

File: src/dto/ParkingZoneDto.js
export class ParkingZoneDto {
    constructor(id, title, address, description, map, parkingSpacesCount, photos) {
      this.id = id;
      this.title = title;
      this.address = address;
      this.description = description;
      this.map = map;
      this.parkingSpacesCount = parkingSpacesCount;
      this.photos = photos; // Массив объектов Photo
    }
  }
  